### DISCUSSION TITLE
Only want to see actions for repos/orgs I am in and people/orgs I am following

### DISCUSSION THREAD

**Discussion by PythonCoderAS** [community/community#46284] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I _do not_ want to see content from unknown users! Please only show me content from repos and orgs I am a member of and users and orgs I am following or at least provide an opt-out.

**Comment by SilasRodrigues19** [comment]
Hey @PythonCoderAS 

Are you referring to the `For You` tab?

**Reply by SilasRodrigues19** [reply]
If so, you can set options on the filter to adjust as you like.

![image](https://user-images.githubusercontent.com/47532703/216643293-1b6dae84-fe98-470a-9a10-549dd44fde6a.png)

**Reply by PythonCoderAS** [reply]
Yes, the "New feed". However, that filter is only for what type of activities I see, not for which user/repo/org it's by. For example, I can set my filter to "releases" only but then I get releases from repos I have not starred or follow the user that made the release/owns the repo.
